src/composables/multisigAccounts.ts

```diff
@@ -76,6 +76,17 @@ export function useMultisigAccounts({ store }: IDefaultComposableOptions) {
     pendingMultisigAccounts.value = getStoredMultisigAccounts(activeNetwork.value.networkId, true);
   }
 
+  if (
+    !activeMultisigAccountId.value
+    || activeMultisigNetworkId.value !== activeNetwork.value.networkId
+  ) {
+    activeMultisigAccountId.value = getLocalStorageItem<string>([
+      LOCAL_STORAGE_MULTISIG_KEY,
+      'active',
+      activeNetwork.value.networkId,
+    ]) || '';
+  }
+
```

Comment:
on this part of code i see 3 times "activeNetwork.value.networkId"

Reply:
I'd prefer not to rewrite whole composable